<launch>

  #### Asctec Autopilot ###################################

  <node pkg="asctec_autopilot" type="autopilot_node" name="AutoPilotNode" output="screen">

    <remap from="/asctec/ESTOP" to="/mav/estop"/>  

    <param name="port" type="string" value="/dev/ttyS3"/>
    <param name="speed" type="int" value="57600"/>
    <param name="freq" type="double" value="5.0"/>

    <param name="enable_IMU_CALCDATA" type="bool" value="true"/>
    <param name="interval_IMU_CALCDATA" type="int" value="1"/>
    <param name="offset_IMU_CALCDATA" type="int" value="0"/>

    <param name="enable_LL_STATUS" type="bool" value="false"/>
    <param name="interval_LL_STATUS" type="int" value="2"/>
    <param name="offset_LL_STATUS" type="int" value="0"/>
    
    <param name="enable_GPS_DATA" type="bool" value="true"/>
    <param name="interval_GPS_DATA" type="int" value="2"/>
    <param name="offset_GPS_DATA" type="int" value="0"/>
  </node>

  <node pkg="rosbag" type="record" name="rosbag_record"
       args="record -o log_imu_data /asctec/IMU_CALCDATA /asctec/GPS_DATA" output="screen"
       />

</launch>
